### DISCUSSION TITLE
Got a waring when using in a framework target.

### DISCUSSION THREAD

**Discussion by bluesky335** [krzyzanowskim/CryptoSwift#940] [General]
**Describe the bug**

After I add CryptoSwift to xcode using swift package manager. Then I import it in a framework target.  I got this warning: 
`Module 'CryptoSwift' was not compiled with library evolution support; using it means binary compatibility for 'MyFrameworkTarget' can't be guaranteed.`

**Reproduce**
Steps to reproduce:
1. Add CryptoSwift to xcode using swift package manager.
2. Create a frame work target. set it's build setting **Build Libraries for Distribution** to YES.
3. import 'CryptoSwift', then got the warning: `Module 'CryptoSwift' was not compiled with library evolution support; using it means binary compatibility for 'xxxx' can't be guaranteed`.

**Comment by krzyzanowskim** [comment]
What version do you integrate?

Generally speaking CryptoSwift with SPM is distributed as source code that is then built by your project and I as a vendor have no control over how it is built, specifically, whether **Build Libraries for Distribution** is on or off. I'd say it depends on your project setup how the dependency built.

Anyway, can you confirm you use 1.4.2?
